data/sql_updates/create_linkage_names_vw.sql

```diff
@@ -5,7 +5,8 @@ select
     dimlinkages.cand_sk as candidate_key,
     dimlinkages.cmte_sk as committee_key,
     dimlinkages.cand_id as candidate_id,
-    dimlinkages.cand_election_yr as active_through,
+    max(dimlinkages.cand_election_yr) as active_through,
```

Comment:
I don't think this is going to work the way you want. With a `group by` on the table's primary key, this will mean that `max(cand_election_yr)` will always equal `cand_election_year`, since grouping by the primary key won't actually be aggregating any rows.